examples/device-a.py

```diff
@@ -10,4 +10,5 @@ def hello(event, data):
 def every(event, data):
     print(event, data)
 
-device.run()
+device.start()
```

Comment:
Is `device.run()` something we want to keep? It just saves one line of typing which is not a big deal.

Reply:
I don't think saving a line is worth it